### PR TITLE
feat: expose list/get options for full/basic on schema methods

### PR DESCRIPTION
**Pull request by feywind**
This corrects a review issue in the unreleased PR: https://github.com/googleapis/nodejs-pubsub/pull/1289

### CODE REVIEW

**Comment by feywind** [comment]
@pradn @hongalex Can y'all verify that this meets the desired functionality? Hoping to get the API finished nailing down before releasing it.

**Review by hongalex — APPROVED** [review]
Looks good, thanks! I'll wait for Prad to confirm

**Comment by feywind** [comment]
Fixing that flaky test is a separate task...

File: src/pubsub.ts [hunk 1]
```diff
@@ -772,23 +771,30 @@ export class PubSub {
...
+   * @returns {AsyncIterable<ISchema>}
    *
    * @example
    * for await (const s of pubsub.listSchemas()) {
    *   const moreInfo = await s.get();
    * }
    */
-  async *listSchemas(options?: CallOptions): AsyncIterable<Schema> {
+  async *listSchemas(
+    view: google.pubsub.v1.SchemaView = SchemaViews.Full,
```
Comment (pradn) [inline src/pubsub.ts:787]: Let's use BASIC as the default for ListSchemas.

FULL should be the default for GetSchemas, as this code already does.

**Review by pradn — CHANGES_REQUESTED** [review]

**Review by pradn — APPROVED** [review]
Perhaps a test for this behavior?

**Pull request merged**